### PR TITLE
psalm to level 2

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**

### CODE REVIEW

**Comment by JellyBellyDev** [comment]
@AlessandroMinoccheri Good Job!!! Please can you fix CI for the php8.0? Thanks!

**Comment by AlessandroMinoccheri** [comment]
@JellyBellyDev I think that we need to upgrade our php80 container using a version different from php8.0.0 and everything should be ok.

**Comment by JellyBellyDev** [comment]
> @JellyBellyDev I think that we need to upgrade our php80 container using a version different from php8.0.0 and everything should be ok.

i upgrade the docker image, please replay the CI! Thanks!

**Review by garak — APPROVED** [review]

File: src/Poser.php [hunk 1]
```diff
@@ -13,12 +13,9 @@ class Poser
      *
      * @param $renders
      */
-    public function __construct($renders)
+    public function __construct(array $renders)
```
Comment (AlessandroMinoccheri) [inline src/Poser.php:16]: This could be a BC.
cc @JellyBellyDev @garak

Comment (garak) [reply src/Poser.php:16]: Who cares? 😜

Comment (JellyBellyDev) [reply src/Poser.php:16]: We can create a v3?

Comment (AlessandroMinoccheri) [reply src/Poser.php:16]: If someone uses this method it's a BC.
I have changed also in the README file how to use this class.

Comment (JellyBellyDev) [reply src/Poser.php:16]: I see! Please can you change also the CHANGELOG.md file?

Comment (AlessandroMinoccheri) [reply src/Poser.php:16]: Done

**Review by JellyBellyDev — APPROVED** [review]
GJ!

**Pull request merged**